src/device/wg_controller.rs

```diff
@@ -54,6 +54,12 @@ pub async fn consolidate_wg_state(
     entities: &Entities,
     features: &Features,
 ) -> Result {
+    if features.post_quantum_vpn.is_some() && requested_state.postquantum_keys.is_none() {
+        // The post quantum state is not ready, we don't want to set up quantum
+        // unsafe tunnel
+        return Ok(());
```

Comment:
If I understand the idea correclty, IMHO, in such a case we should still consolidate, but against empty list of WG peers.

This `if` now basically means, that
```
if PQ is enabled, but keys are not yet established, then please leave the WG configuration unchanged
```

And I believe we would like to clear any WG peers in such a case? Or am I mistaken?

Reply:
I fixed this one by not adding a peer to the list when it's set as an exit node but keys are not available yet. The rest of the consolidation is proceeding as usual